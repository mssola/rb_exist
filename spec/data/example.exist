if (xmldb:collection-available('_{source}'))
then
    (: The source is a collection name :)
    let $status := xmldb:copy('_{source}', '_{dest}')
    return <status>{$status}</status>
else
    (: The source is a resource name :)
    let $status := xmldb:copy('_{srccol}', '_{dest}', '_{srcres}')
    return <status>{$status}</status>
