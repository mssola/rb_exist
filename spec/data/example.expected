if (xmldb:collection-available('source'))
then
    (: The source is a collection name :)
    let $status := xmldb:copy('source', 'dest')
    return <status>{$status}</status>
else
    (: The source is a resource name :)
    let $status := xmldb:copy('src_col', 'dest', 'src_res')
    return <status>{$status}</status>